--- 727-230.xml	2013-05-16 07:01:44.875800306 -0700
+++ 727-230.xml.mod	2013-05-16 07:06:16.576806504 -0700
@@ -170,9 +170,11 @@
 	<control-speed control="EXTEND" transition-time="7"/>
   </gear>
 
-
-<!--tank-->
-  <tank x="6" y="0" z="0.2" jet="true" capacity="22212"/>
+  <!-- tanks  -->
+  <!--IAHMCOL: Location of tanks might need to be refined-->
+  <tank x="4.084" y="2.098" z="-0.6806" jet="true" capacity="12700"/>
+  <tank x="6" y="0" z="0.2" jet="true" capacity="28963"/>
+  <tank x="8.084" y="2.098" z="-0.6806" jet="true" capacity="12700"/>
 
     <ballast x="8.5" y="0" z="0.2" mass="9000"/>
 
